<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewPerishable</class>
 <widget class="QWidget" name="AddNewPerishable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Perishable Product</string>
  </property>
  <widget class="QLabel" name="label_Heading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add A New Perishable Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="50,100,50">
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Category">
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Name"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Manu"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Price">
      <property name="text">
       <string>Unit Price</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Category"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Manufacturer">
      <property name="text">
       <string>Manufacturer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Price"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Stock">
      <property name="text">
       <string>Initial Stock</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Barcode">
      <property name="text">
       <string>Barcode</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_Expiry">
      <property name="text">
       <string>Expiry Date</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Expiry">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>yyyy-mm-dd</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_Stock"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Barcode"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="error_Name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="error_Barcode">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="error_Category">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="error_Manu">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="error_Price">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="error_Stock">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="error_Expiry">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QPushButton" name="create">
      <property name="text">
       <string>Create New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddNewPerishable</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
